vspreview/core/types/video.py

```diff
@@ -172,10 +172,10 @@ class ChromaLoc:
         'frame_to_show', 'scening_lists'
     )
     __slots__ = storable_attrs + (
-        'index', 'width', 'height', 'fps_num', 'fps_den',
-        'total_frames', 'total_time', 'graphics_scene_item',
```

Comment:
Please, do try to keep the changes to a minimum.
Especially for reformatting of unrelated stuff.